config/index.js

```diff
@@ -4,6 +4,26 @@
 
 const path = require('path')
 
+// get the browser command, specifically google chrome
+const browserCommand = getBrowserCommand()
+
+// return browser command for the given operating system
+function getBrowserCommand () {
+  // get the operating system version that we are running on
+  const platform = process.platform
+  if (platform === 'darwin') {// if we are running on mac os
+    return 'Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome'
+  } else if (platform === 'win32'){ // if we are running on windows
```

Comment:
what about win 64?! 😜

Reply:
All windows platforms are given the word `win32` under NodeJS [Process.platform](https://nodejs.org/api/process.html#process_process_platform) . Python for example, calls all windows platforms `nt`